parkhangs-webapp/src/components/ShareCalendar.js

```diff
@@ -13,11 +14,19 @@ class ShareCalendar extends React.Component {
   }
  
   render() {
-    let icon = { 'calendar-plus-o': 'left' };
+    let icon= { 'calendar-plus-o': 'left' };
+    let items= [ 
+      {google: 'Google'},
```

Comment:
we dont want the other ones (like Outlook, Yahoo?)

Reply:
I can add them back in if we still want them, but I thought the drop down would be cleaner with fewer options. Thought last time someone implied we don't need all the options.

Reply:
haha I think that was me. Em up to you, we can always add them back on later but it does't matter for now.